src/layouts/DrawerContent.tsx

```diff
@@ -86,6 +100,33 @@ const DrawerContent = (props: DrawerContentProps & DrawerItemListProps) => {
     currentDrawerItem = history.pop();
   }
 
+  const onPressHelp = () => {
+    props.fetchHelp(helpRequestForMobile);
+  };
+
+  const navigateHelp = () => {
+    let url = helpConfig?.redirectUrls[0]?.url;
+    if (url === undefined) {
+      url = helpConfig?.defaultRedirectUrl;
+    }
+    if (url !== undefined) {
+      Linking.canOpenURL(url).then((supported) => {
+        if (supported) {
+          if (url !== undefined) {
```

Comment:
Do we need to check this again? in line 112 also checked

Reply:
No need at run time. Did that just for avoiding compile-time error with passing undefined value for OpenURL.

Reply:
No need at run time. Did that just for avoiding compile-time error with passing undefined value for OpenURL.